website/versioned_docs/version-29.5/TestingAsyncCode.md

```diff
@@ -126,8 +126,8 @@ Jest will throw an error, if the same test function is passed a `done()` callbac
 You can also use the `.resolves` matcher in your expect statement, and Jest will wait for that promise to resolve. If the promise is rejected, the test will automatically fail.
 
 ```js
-test('the data is peanut butter', () => {
-  return expect(fetchData()).resolves.toBe('peanut butter');
+test('the data is peanut butter', async () => {
+  await expect(fetchData()).resolves.toBe('peanut butter');
```

Comment:
Just below the text says: "Be sure to return the assertion..." I guess that should be reworked as well.